splink/__init__.py

```diff
@@ -98,8 +99,8 @@ def manually_apply_fellegi_sunter_weights(self):
 
     def get_scored_comparisons(self, num_iterations=None):
 
-        if (num_iterations is None):
-            num_iterations=self.settings["max_iterations"]
+        if not num_iterations:
```

Comment:
Not relevant to this PR, I'm just cleaning up code here.  Specifically, the number of iterations is taken from the settings object unless explicitly set.

Reply:
Ah, I wasn't sure whether `None` could be coerced to a boolean, hence treating it like a value to be sure without having to look it up. Good to know.